py-server/history_tutor/history_tutor.py

```diff
@@ -67,10 +66,10 @@
         {{
             "question": "What were the different theories about the cause of the Black Death?",
             "answers": [
-            {{"answer": "Religion: God sent the plague as a punishment for people's sins.", "hasAnswered": false}},
-            {{"answer": "Miasma: bad air or smells caused by decaying rubbish.", "hasAnswered": false}},
-            {{"answer": "Four Humours: most physicians believed that disease was caused by an imbalance in the Four Humours.", "hasAnswered": false}},
-            {{"answer": "Outsiders: strangers or witches had caused the disease.", "hasAnswered": false}}
+            {{"answer": "Religion: God sent the plague as a punishment for people's sins.", "hasAnswered": "false"}},
+            {{"answer": "Miasma: bad air or smells caused by decaying rubbish.", "hasAnswered": "false"}},
```

Comment:
I believe that providng detailed examples may risk overfitting the model's generated questions. I have been experimenting with only providing the (typescript) interfaces to the model, without examples, and it seems to work fine.

Are you seeing value in providing few-shot?